README.md

```diff
@@ -1,2 +1,185 @@
-# microverse-finMarket
-A web-app to get updated financial statements in real time, every statements is audited, standardized, and up to date. It cover NYSE, NASDAQ, AMEX, EURONEXT, TSX, INDEXES, ETFs, MUTUAL FUNDS, and FOREX.
+<a name="readme-top"></a>
+
+  <h3><b> Financial Market Capstone project</b></h3>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run Linters](#run-linters)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 Financial Market Capstone project <a name="about-project"></a>
+
+Financial Market Capstone project developed with React, gitflow, and githubflow. In addition, i added Linters for Css and Javascript.
+
+## 🛠 Built With <a name="built-with"></a>
+
+1-Reactjs
+2-Css
+3-Js
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://create-react-app.dev/">REACT</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">JS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Initial setup with CRA.
+- Add Redux-toolkits.
+- Fetch data from APIs.
+- Render Data.
+- Add Linters.
+- Add Unit testing
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE LINK -->
+
+## Live Demo <a name="live-demo"></a>
+
+[Lve Demo Link](https://finmarket.netlify.app/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Your video description is great @Errytagedesign, I notice you have mentioned part of the project you are proud of, but did not show pis of the code. Please remember, that you are required to show a highlight about a piece of the code that you find interesting or you are proud of making. This could be achieved by adding your VS-Code to the video description with the code you find interesting.